server/src/actions/getSingleConnectedUser.ts

```diff
@@ -0,0 +1,11 @@
+import { connectedUsersCollection } from '../utilities/firebaseInit'
+
```

Comment:
You can delete this file and move all of this into getConnectedUsers!

Reply:
Done